src/components/SeriesNav/SeriesNav.js

```diff
@@ -0,0 +1,179 @@
+
+function SeriesNav({
+  documentId,
+  series,
+  inline,
+  ActionBar,
+  Link = DefaultLink,
+  PayNote,
+  onEpisodeClick
+}) {
+  const [colorScheme] = useColorContext()
+
+  const currentTile =
+    documentId &&
+    series.episodes.find(episode => episode.document?.id === documentId)
+  const currentTileIndex = series.episodes.indexOf(currentTile)
+
+  const localColors = {
+    light: {
+      default: '#191919',
+      text: '#f0f0f0'
+    },
+    dark: {
+      default: '#ffffff',
+      text: '#282828'
+    }
+  }
+
+  // add paynote object after current episode or to third card if no current episode
+  const payNoteObject = { isPayNote: true }
+  const payNotePosition = currentTile
+    ? Math.max(currentTileIndex + 1, DEFAULT_PAYNOTE_INDEX)
+    : DEFAULT_PAYNOTE_INDEX
+  const episodes =
+    PayNote && !inline
+      ? [
+          ...series.episodes.slice(0, payNotePosition),
+          payNoteObject,
+          ...series.episodes.slice(payNotePosition)
+        ]
+      : [...series.episodes]
+
+  return (
+    <div {...(inline ? styles.containerInline : styles.container)}>
+      {inline ? (
+        <Center>
+          <hr
+            {...styles.hline}
+            {...colorScheme.set('borderColor', 'divider')}
+          />
+          {series.overview?.meta?.path ? (
+            <Link href={series.overview.meta.path} passHref>
+              <a {...styles.plainlink}>
+                <h3 {...styles.title}>{series.title}</h3>
+              </a>
+            </Link>
+          ) : (
+            <h3 {...styles.title}>{series.title}</h3>
+          )}
+          <p {...styles.description}>{series.description}</p>
+        </Center>
+      ) : null}
+
+      <TeaserCarousel
+        grid={!inline}
+        style={{
+          padding: 0,
+          backgroundColor: 'transparent'
+        }}
+      >
+        <TeaserCarouselTileContainer
+          initialScrollTileIndex={currentTileIndex}
+          style={{
+            padding: 0
+          }}
+          overflowCentered
+        >
+          {episodes.map(episode => {
+            return (
+              <SeriesNavTile
+                key={episode.title}
+                PayNote={PayNote && episode.isPayNote && PayNote}
+                current={documentId && documentId === episode?.document?.id}
+                episode={episode}
+                inline={inline}
+                ActionBar={ActionBar}
+                Link={Link}
+                onEpisodeClick={onEpisodeClick}
+              />
+            )
+          })}
+        </TeaserCarouselTileContainer>
+      </TeaserCarousel>
+
+      {inline ? (
+        <ColorContextLocalExtension localColors={localColors}>
+          {PayNote ? (
+            <PayNote inline={inline} />
+          ) : (
+            <hr
+              {...styles.hline}
+              {...colorScheme.set('borderColor', 'divider')}
+            />
+          )}
+        </ColorContextLocalExtension>
```

Comment:
Let's minimise the `ColorContextLocalExtension` use to only when necessary:

```suggestion
        {PayNote ? (
          <ColorContextLocalExtension localColors={localColors}>
            <PayNote inline={inline} />
          </ColorContextLocalExtension>
        ) : (
          <hr
            {...styles.hline}
            {...colorScheme.set('borderColor', 'divider')}
          />
        )}
```